build.gradle

```diff
@@ -37,6 +38,8 @@ dependencies {
 	testImplementation 'org.springframework.security:spring-security-test'
 	testImplementation 'org.codehaus.groovy:groovy:3.0.18'
 	testImplementation 'org.spockframework:spock-core:2.0-groovy-3.0'
+	implementation 'com.diffplug.spotless:spotless-plugin-gradle:6.20.0'
```

Comment:
spotless is a plugin and not a lib with scope `implementation`, we don't need it to run de app :eyes: 
https://plugins.gradle.org/plugin/com.diffplug.gradle.spotless